modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -1458,7 +1458,7 @@ definition {
 
 		ObjectPortlet.viewEntryDetails(entry = "Entry A");
 
-		CreateObject.deleteEntryInRelationshipTab(labelName = "Entry B");
+		CreateObject.deleteEntryInRelationshipTab(labelName = "Entry A");
```

Comment:
Hey @larissacribeiro, can you explain what motivated this change?

Do you believe this change would be beneficial when we think about what is being tested?

Reply:
Hi @igor-franca, this test was failing locally master, so I checked that the entry it is trying to delete is A and not B, that's the reason. If you check it's [history](https://testray.liferay.com/#/project/35392/routines/45357/build/77098253/case-result/77153878/history?filter=%7B%22testrayRoutineIds%22%3A%5B45357%5D%7D&filterSchema=buildResultsHistory) you will see that it has failled other times in other pr's, and in the poshi report the entry it fails to delete is A.